train_management/models/abstract_dvzo_model.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from django_currentuser.db.models import CurrentUserField
+
+
+class AbstractDvzoModel(models.Model):
```

Comment:
That's not an abstract class right? :)

Reply:
it's an abstract model

Reply:
Dass ist nicht ganz korrekt. Es ist vielleicht eher ein dvzo Basis Model, weil es von einer konkreten Implementation erbt, die selbst nicht abstrakt ist (models.Model).
Aber du kannst das Naming auch so belassen, wenn du möchtest.